src/App.tsx

```diff
@@ -1,16 +1,23 @@
 import './App.scss';
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+  const movieAdd = (newMovie: Movie) => {
```

Comment:
```suggestion
  const addMovie = (newMovie: Movie) => {
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/e1686a28-49bb-4dc7-9322-4da376d4eb4e)